source/extensions/filters/common/ext_authz/ext_authz_http_impl.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "envoy/upstream/cluster_manager.h"
+
+#include "extensions/filters/common/ext_authz/ext_authz.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Filters {
+namespace Common {
+namespace ExtAuthz {
+
+class RawHttpClientImpl : public Client, public Http::AsyncClient::Callbacks {
```

Comment:
Can you add comments for this and the gRPC variant explaining what it is each one does specifically and why we have two variants?